s4/clarity/_internal/factory.py

```diff
@@ -43,32 +43,73 @@ class ElementFactory(object):
     def _strip_params(string):
         return ElementFactory._params_re.sub('', string)
 
-    def __init__(self, lims, element_class, batch_flags=None, request_path=None, name_attribute="name"):
+    def __init__(self, lims, element_class):
         """
+        Creates a new factory to provide interface between Clarity LIMS and the client software.
+        The provided `element_class` will be used to convert the XML records into Python objects
+        and back. There are several class attributes that can be set to configure this interface.
+
+        NAME_ATTRIBUTE: This is used if the name of the class differs from the name in the Clarity API.
+
+        REQUEST_PATH: If the class has the REQUEST_PATH attribute then it will
+        be used to override the default path. It is used in relation to the root URI of the Clarity API.
```

Comment:
The help has become less useful in that it doesn't describe what the default is.